### PR TITLE
Adds Targeting to Some Emotes, Cleans up Emotes Backend

### PR DESCRIPTION
**Pull request by KasparoVy**
You can wink, buzz2, scream, smile, shake head, wave, nod and snap fingers at people now.

emotesworking
voxshriekat

Someone mentioned something on the Discord and I just decided to go for it. 🤷‍

### CODE REVIEW

**Comment by TravellingMerchant** [comment]
Nice!

No reason to object to this. It's good!

**Comment by KasparoVy** [comment]
Gonna be another commit to come, few mins

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -930,7 +959,7 @@
 
 
 
-	if(message) //Humans are special fucking snowflakes and have 735 lines of emotes, they get to handle their own emotes, not call the parent
+	if(message) //Humans are special fucking snowflakes and have 1000 lines of emotes, they get to handle their own emotes, not call the parent
```
Comment (SamHPurp) [inline code/modules/mob/living/carbon/human/emote.dm]: 😆

Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/emote.dm]: Hopefully less once I'm done, commit soon

**Comment by KasparoVy** [comment]
There we go, that's more like it.

**Pull request merged**